clients/roscpp/include/ros/service_client.h

```diff
@@ -199,6 +206,7 @@ class ROSCPP_DECL ServiceClient
     M_string header_values_;
     std::string service_md5sum_;
     bool is_shutdown_;
+    double timeout;		///< Timeout for calling service
```

Comment:
should probably mention that unit is seconds

Reply:
Changed it to ros::WallDuration, so no need for that anymore.